CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 2.3.4-dev
```

Comment:
[nit] We recently changed the guidance to use `-wip` as a suffix. `-dev` can cause some confusion because there are legitimate workflows (that the Dar team usually doesn't use) for publishing with a `-dev` suffix, while `-wip` is more clearly not intended for publish.

Reply:
SGTM. Done.